src/main/java/core/basesyntax/model/TypeOfFruit.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.model;
+
+public enum TypeOfFruit {
+    BANANA, APPLE
+}

```

Comment:
Please remove, we cannot know which fruits will be in input file. Probably grapes, tomatoes or anything else.
We shouldn't stick to any names